Run.scala

```diff
@@ -0,0 +1,13 @@
+object Run {
+   implicit class IntTimes(x: Int) {
+      def times [A](f: =>A): Unit = {
+         def loop(current: Int): Unit =
+         
+         if(current > 0){
+            f
```

Comment:
remove unnecessary variable call

Reply:
No, that's wrong

Reply:
https://www.youtube.com/watch?v=e53HfowZCdI